src/GEG_LULUCF_Carbon_BI_VBI1&2.Rmd

```diff
@@ -168,123 +189,683 @@ Resultaat <- Resultaat %>%
+
+
+### Aparte schatting gepaarde en niet-gepaarde plots
+
++ Voor de gepaarde metingen kunnen we op plotniveau de toename in C per ha per jaar bepalen. Dit is correcter gezien de periode tussen de herhaalde metingen varieert van plot tot plot. 
+
+```{r}
+results_paired <- paired_p1_p2 %>%
+  mutate(Weight = Weight_2,
+         Year = Year_2,
+         Reeks = 1) %>%
+  My.WgtParEstimation(VariableName =  c("carbon_t_ha_diff_year", "carbon_t_ha_diff_total"))
+
+```
+
++ voor de niet-gepaarde metingen berekenen we het verschil in gemiddelde houtvoorraad periode en delen we die door het verschil in gemiddelde jaar waarin de opname is gebeurd.
+
+
+```{r}
+results_unpaired_periode1 <- unpaired %>%
+  My.WgtParEstimation(VariableName =  "Carbon_t_ha", Periode = 1)
+
+results_unpaired_periode2 <- unpaired %>%
+  My.WgtParEstimation(VariableName =  "Carbon_t_ha", Periode = 2)
+
+periode_jaar_unpaired <- unpaired %>%
+  group_by(Periode) %>%
+  summarise(year_mean = mean(Year),
+            year_middle = (max(Year) - min(Year))/2 + min(Year)) %>%
+  ungroup() %>%
+  pivot_longer(cols = c("year_mean", "year_middle"), names_to = "type", values_to = "year") %>%
+  pivot_wider(names_from = "Periode", names_prefix = "periode_", values_from = c("year")) %>%
+  mutate(diff_n_years = periode_2 - periode_1)
+
+results_unpaired <- results_unpaired_periode1 %>%
+  bind_rows(results_unpaired_periode2) 
+
+colnames(results_unpaired) <- str_to_lower(colnames(results_unpaired))
+
+results_unpaired_wide <- results_unpaired %>%
+  select(variabele, periode, nbobservaties, mean = wgt.mean, se, llci, ulci) %>%
+  pivot_wider(names_from = "periode", values_from = c("nbobservaties", "mean", "se", "llci", "ulci")) %>%
+  bind_cols(periode_jaar_unpaired) %>%
+  mutate(
+    verschil = mean_2 - mean_1,
+    verschil_se = sqrt(se_1^2 + se_2^2),
+    llci = verschil - 1.96 * verschil_se,
+    ulci = verschil + 1.96 * verschil_se,
+    verschil_jaar = verschil/diff_n_years,
+    verschil_jaar_se = verschil_se/diff_n_years,
+    verschil_jaar_llci = llci/diff_n_years,
+    verschil_jaar_ulci = ulci/diff_n_years,
+    variabele = "carbon_t_ha_diff_year"
+  )
+```
+
+Dit geeft volgende resultaten:
+
+```{r}
+results_paired_summary <- results_paired %>%
+  mutate(observation_type = "gepaard",
+         verschil_jaar = str_c(round(wgt.mean, 2), " [", round(llci, 2), " - ", round(ulci, 2), "]")) %>%
+  select(variabele, observation_type,  "Jaarlijkse toename" = verschil_jaar) 
+  
+results_unpaired_summary <- results_unpaired_wide %>%
+  mutate(observation_type = "ongepaard",
+         diff_n_years = round(diff_n_years, 2),
+         verschil_jaar = str_c(round(verschil_jaar, 2), " [", round(verschil_jaar_llci, 2), " - ", round(verschil_jaar_ulci, 2), "]")) %>%
+  select(variabele, observation_type, "Tijdsverschil (jaren)" = diff_n_years,  "Jaarlijkse toename" = verschil_jaar) 
+
+results_paired_summary %>%
+  bind_rows(results_unpaired_summary) %>%
+  datatable(rownames = FALSE)
+
```

Comment:
In deze tabel wordt voor gepaard diff_year en diff_total getoond, maar voor ongepaard niet.